pkg/util/util.go

```diff
@@ -73,6 +72,28 @@ func NewGCPSecretCR(namespace, creds string) *corev1.Secret {
 	}
 }
 
+// GetGCPParentFolderFromConfigMap returns a configmap value if it's exist
+func GetGCPParentFolderFromConfigMap(kubeClient kubeclientpkg.Client, namespace, name string) (string, error) {
+	var orgParentFolderID = "240634451310" // Service Delivery hard coded org subfolder
+	configmap := &corev1.ConfigMap{}
+	err := kubeClient.Get(context.TODO(),
+		kubetypes.NamespacedName{
+			Namespace: namespace,
+			Name:      name,
+		},
+		configmap)
+	if err != nil {
+		return orgParentFolderID, fmt.Errorf("clusterdeployment.GetGCPParentFolderFromConfigMap.Get %v", err)
+	}
+	orgParentFolderIDconfig, ok := configmap.Data["orgParentFolderID"]
+	if !ok {
+		return orgParentFolderID, fmt.Errorf("GCP config %v did not contain key %v",
+			name, "orgParentFolderID")
+	}
+
+	return orgParentFolderIDconfig, nil
```

Comment:
For now we only need the parent folder, so this function is ok for now, but sooner or later we will read more values from that config map and then we will want a config struct to be filled. If you want, I think you can already create it and return it here instead of the single value.

Reply:
@NautiluX makes sense, I'll modify here